frontend/src/metabase/static-viz/components/XYChart/utils/series.unit.spec.ts

```diff
@@ -103,40 +103,39 @@ describe("sortSeries", () => {
 });
 
 describe("calculateStackedItems", () => {
-  const series: Series[] = [
```

Comment:
move this variable inside its own test since it isn't relevant to the new test I added.